File: Array/Merge.py
def merge(arr1, arr2):
    m = len(arr1)
    n = len(arr2)
    k = 0
    arr3 = [0] * (m+n)
    for i in range(m):
        arr3[k] = arr1[i]
        k += 1
    for j in range(n):
        arr3[k] = arr2[j]
        k += 1
    return arr3


m = int(input('Enter size of Array 1: '))
print(f'Enter {m} elements:')
arr1 = list(map(int, input().split()))
n = int(input('Enter size of Array 2: '))
print(f'Enter {n} elements:')
arr2 = list(map(int, input().split()))
print('Merged Array is:')
res = merge(arr1, arr2)
print(res)

File: Array/Merge.java
package Array;

public class Merge {
  
  public static int[] mergeArrays(int[] a, int[] b) {
    int[] c = new int[a.length + b.length];
    int k = 0;
    for (int i = 0; i < a.length; i++) {
      c[k] = a[i];
      k++;
    }
    for (int i = 0; i < b.length; i++) {
      c[k] = b[i];
      k++;
    }
    return c;
  }

  public static void main(String[] args) {
    int[] a = Basic.inputArray();
    int[] b = Basic.inputArray();
    int[] c = mergeArrays(a, b);
    System.out.println("Merged Array is:");
    Basic.printArray(c);
  }
}
